GitStartHQ/graphql-engine
# [GQL-25] #986 Feature request: Untrack multiple tables at once on the data page

### Description


The process to untrack tables is very tedious currently. One has to navigate each table , click on Modify tab and untrack each table. This PR shows all the tracked tables in the data page and makes it easy to untrack tables directly from there with a single click of a button
### Changelog


### Affected components



### Related Issues



#986
### Solution and Design


In order to solve this tedious process of untracking tables. This PR presents the list of all tracked tables in the data page and provides a way to untrack the tables directly from the data page with a click of a button.

https://user-images.githubusercontent.com/59878660/177365691-92f4788e-23aa-4aba-a940-cc85acc131fc.mp4


### Steps to test and verify


1. Go to schema page 
2. See the list of tracked tables with untrack buttons
3. Click on untrack button to untrack the table
4. Observe the result in the page

### Server checklist


#### Catalog upgrade

Does this PR change Hasura Catalog version?

#### Metadata


Does this PR add a new Metadata feature?
  - Does `run_sql` auto manages the new metadata through schema diffing?
  - Does `run_sql` auto manages the definitions of metadata on renaming?
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?


#### GraphQL
   
   

#### Breaking changes


  1. Metadata API

     Existing `query` types:
     

  5. GraphQL API

     Schema Generation:
     
     
     

     Schema Resolve:-
     
     

  6. Logging
